### PR TITLE
Run app and tests in Docker

### LINKED ISSUE #14: Add running the app in Docker

### PR DESCRIPTION
**Pull request by jedr**

### CODE REVIEW

**Comment by jedr** [comment]
Chcę dorzucić commit ale chyba nie mogę 😭 

```console
$ git push
Enumerating objects: 5, done.
Counting objects: 100% (5/5), done.
Delta compression using up to 4 threads
Compressing objects: 100% (3/3), done.
Writing objects: 100% (3/3), 308 bytes | 308.00 KiB/s, done.
Total 3 (delta 2), reused 0 (delta 0)
remote: Resolving deltas: 100% (2/2), completed with 2 local objects.
remote: error: GH006: Protected branch update failed for refs/heads/run-in-docker.
remote: error: Waiting on code owner review from tqZK.
To github.com:tqZK/harc_game_web.git
 ! [remote rejected] run-in-docker -> run-in-docker (protected branch hook declined)
error: failed to push some refs to 'git@github.com:tqZK/harc_game_web.git'
```

**Comment by jedr** [comment]
Już mogę 👍 🙂

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.9.0
+WORKDIR /app
+COPY requirements.txt ./
+RUN pip install -r requirements.txt
+COPY . .
+RUN ["python", "harc_game_web/manage.py", "makemigrations"]
+RUN ["python", "harc_game_web/manage.py", "migrate"]
+RUN ["python", "harc_game_web/manage.py", "loaddata", "example_db.json"]
+CMD ["python", "harc_game_web/manage.py", "runserver", "0.0.0.0:8000"]
```
Comment (zkunik) [inline Dockerfile]: could we use `make`s from Makefile?

**Review by zkunik — APPROVED** [review]
🐳 🐋

File: Dockerfile (hunk 1 above)
Comment (jedr) [reply Dockerfile]: Surely that would be nice! We would need to have `make` in the Docker container though, which I don't think exists in the `python` image (need to check).

Comment (jedr) [reply Dockerfile]: Actually it looks like `make` is available, so let me have another look at it. Thanks @tqZK for the suggestion! 👏

**Comment by jedr** [comment]
@tqZK can you have another look? I've adjusted Dockerfile to use `make`, this looks much cleaner. 👍

**Comment by zkunik** [comment]
@jedr sorry for the inconvenience, I changed default branch from `master` to `main` and deleted `master` branch. Please open new PR to `main` branch.

**Comment by tqZK** [comment]
@jedr sorry for the inconvenience, I changed default branch from `master` to `main` and deleted `master` branch. Please open new PR to `main` branch.

**Review by zkunik — COMMENTED** [review]
Usage of `make` looks great :)

**Pull request closed without merging**